### PR TITLE
Add admin panel login

### PR DESCRIPTION
**Pull request by AndriiDruzhytskyi**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -8,9 +8,12 @@ const cors = require('cors');
 
 const indexRouter = require('./routes/index');
 const apiRouter = require('./routes/api');
+const adminApiRouter = require('./routes/adminApi');
 const authRoutes = require('./routes/auth');
+const adminAuthRoutes = require('./routes/adminAuth');
 
 const auth = require('./middlewares/auth');
+const admin_auth = require('./middlewares/adminAuth');
```
Comment (DMTRIP) [inline app.js]: Change your variable name to camelCase style

**Review by Dmitry528 — COMMENTED** [review]
Try 1) Npm run lint 
and fix problems, code-style is important for us

File: app.js (hunk 1 above)
Comment (AndriiDruzhytskyi) [reply app.js]: Fixed.

**Comment by AndriiDruzhytskyi** [comment]
> Try 1) Npm run lint
> and fix problems, code-style is important for us

Fixed some warnings. We should discuss this later.

**Review by Dmitry528 — APPROVED** [review]

**Review by victorpoliushko — APPROVED** [review]

**Review by marrybbb — APPROVED** [review]

**Review by innomaxx — APPROVED** [review]

**Pull request merged**